samples/append_rows_table_to_proto2.js

```diff
@@ -35,26 +34,24 @@ function main(
     const destinationTable = `projects/${projectId}/datasets/${datasetId}/tables/${tableId}`;
     const streamType = managedwriter.PendingStream;
     const writeClient = new WriterClient({projectId: projectId});
-    const bigquery = new BigQuery({projectId: projectId});
 
     try {
-      const dataset = bigquery.dataset(datasetId);
-      const table = await dataset.table(tableId);
-      const [metadata] = await table.getMetadata();
-      const {schema} = metadata;
-      const storageSchema =
-        adapt.convertBigQuerySchemaToStorageTableSchema(schema);
-      const protoDescriptor = adapt.convertStorageSchemaToProto2Descriptor(
-        storageSchema,
-        'root'
-      );
-
       const streamId = await writeClient.createWriteStream({
         streamType,
         destinationTable,
       });
       console.log(`Stream created: ${streamId}`);
 
+      // Get table schema from WriteStream metadata.
+      const writeStream = await writeClient.getWriteStream({
```

Comment:
Same here, if it is pending then you need to call CreateWriteStream, which returns a table schema.